ontology/mondo.md

```diff
@@ -51,6 +51,8 @@ usages:
       - url: https://monarchinitiative.org/phenotype/HP:0001300#disease
         description: "Parkinsonism: Characteristic neurologic anomaly resulting form degeneration of dopamine-generating cells in the substantia nigra, a region of the midbrain, characterized clinically by shaking, rigidity, slowness of movement and difficulty with walking and gait."
     publications:
+      - id: https://medrxiv.org/cgi/content/short/2022.04.13.22273750v1
```

Comment:
```suggestion
      - id: https://www.medrxiv.org/content/10.1101/2022.04.13.22273750
```

Reply:
Btw it looks like this should actually be a top-level entry and not inside the usage